en/docs/tutorials/scenarios/scenario10-notifications-webhooks.md

```diff
@@ -1,4 +1,62 @@
 # Scenario 10 - Notifications Using Webhooks
 
-!!! Note
-    Content to be added.
\ No newline at end of file
+## User Story
+
+**From time to time GoGo transit provides notifications to the Train companies. (For eg: platform unavailability, revised fees etc). GoGo transit is planning to provide these information as webhooks so that the Train companies can subscribe to these notifications without continuously polling.**
+
+WebHooks only allows one-way communication, from a caller web app to calle web app. The client who intends 
+to receive the events from the server/web app/publisher has to register its URL against the interesting events 
+in the publisher. When an event occurs, if a client is registered for that event the publisher will make an 
+HTTP POST request to the registered URL of the client to notify the event.
+
+<img src="{{base_path}}/assets/img/tutorials/scenarios/scenario_notification_api.png" alt="Webhook notifications" title="Webhook notifications" width="60%" />
+
+You can view Webhook API created in WSO2 API manager by going to the publisher 
+(Super tenant - admin). [https://localhost:9443/publisher/apis](https://localhost:9443/publisher/apis) 
+
+You can publish notifications under different topics.
+
+<img src="{{base_path}}/assets/img/tutorials/scenarios/notification_api_topics.png" alt="Notification Topics" title="Notification Topics" width="60%" />
+
+To subscribe to the API, you need to go to the developer portal.
+
+1. Go to [https://localhost:9443/devportal/](https://localhost:9443/devportal/) Dev portal and select **Carbon.super** tenant domain. This will redirect you to the super tenant developer portal.
```

Comment:
```suggestion
1. Go to [https://localhost:9443/devportal/](https://localhost:9443/devportal/) Developer Portal and select **Carbon.super** tenant domain. This will redirect you to the super tenant Developer Portal.
```
Let's use title case for all major components like Developer Portal, Gateway, Key Manager, etc.